.env.example

```diff
@@ -39,6 +39,8 @@ S3_URL=yourS3Url
 NORTHSTAR_URL=https://northstar-qa.dosomething.org
 NORTHSTAR_AUTH_ID=
 NORTHSTAR_AUTH_SECRET=
+NORTHSTAR_CLIENT_SECRET=
+NORTHSTAR_CLIENT_ID=
```

Comment:
@sbsmith86 We used to have this but it was removed [here.](https://github.com/DoSomething/rogue/pull/351/files) Do you know why we need to bring it back?

Reply:
Yea after this PR https://github.com/DoSomething/rogue/pull/403, we now should use client authentication to specify that the user that is making the request is Rogue (the application, not the user logged into the application) so we actually need these now.

Reply:
And this parallel Gateway PR https://github.com/DoSomething/gateway/pull/83

Reply:
@sbsmith86 thanks for the context! makes sense to me 😄